src/ui/View.js

```diff
@@ -147,8 +147,36 @@ exports = class View extends IView {
 
     this.__view._view = this;
 
+    this._tick = null;
+
     this.updateOpts(opts);
   }
+
+  get tick () {
+    // this._tick can be null while "_tick" can exist on the protoype
+    return this._tick || this.__proto__._tick;
```

Comment:
This line really looks like a hack and I am not a big fan of it but I could not find any other way to make this work without changing the current API.
N.B the tick method can either be on the prototype or a regular object property.